### PR TITLE
updating to SA node model.

### PR DESCRIPTION
**Pull request by nickerso**
Switching to the Fabbri et al (2017) SA node cell model and adding ability for stellate and vagus stimulation. Things give the range of behaviour that we are after, but pushing the model beyond what it was intended to be used for.

I've updated the `Dockerfile` but the `entrypoint.sh` will need more work to get it handling the expected input arguments.

### CODE REVIEW

**Comment by nickerso** [comment]
Just to note that a description of this use case is available here: https://docs.google.com/document/d/1jRAqkGqcvtPZJWyVFIweTWhztsFv9Ea3jNSZM_gMSxE/edit?usp=sharing

**Comment by nickerso** [comment]
Did we want to add the feature to give the heart rate?

File: run_model.py [hunk 1]
```diff
@@ -4,26 +4,35 @@
...
+    #s = OpenCOR.openSimulation('/home/opencor/models/action-potential.xml')
+    s = OpenCOR.openSimulation('/home/opencor/models/HumanSAN_Fabbri_Fantini_Wilders_Severi_2017.sedml')
 
     d = s.data()
 
     # Set integration range
-    d.setPointInterval(10)  # ms
-    d.setEndingPoint(100000)  # ms
+    d.setPointInterval(0.01)  # s
+    d.setEndingPoint(10)  # s
```
Comment (hsorby) [inline run_model.py]: This should be in the sedml file.

File: run_model.py [hunk 2]
```diff
@@ -4,26 +4,35 @@
...
 
-    # print('Setting stimulus period to:', stimulus_period)
+    #print('Setting stimulation mode to:', stimulation_mode)
+    #print('Setting stimulation level to:', stimulation_level)
     c = d.constants()
-    c['membrane/period'] = stimulus_period  # ms
+    c['Rate_modulation_experiments/Iso_1_uM'] = 1.0  # dimensionless
+    if stimulation_mode == 1:
+        # Stellate stimulation 0 - 1 :: 22 - 0
```
Comment (hsorby) [inline run_model.py:16]: Stellate simulation speeds the heart rate up right.

File: HumanSAN_Fabbri_Fantini_Wilders_Severi_2017.sedml [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+            <listOfVariables>
+                <variable id="yVariable1_1" target="/cellml:model/cellml:component[@name='Membrane']/cellml:variable[@name='V']" taskReference="repeatedTask"/>
+            </listOfVariables>
+            <math xmlns="http://www.w3.org/1998/Math/MathML">
+                <ci> yVariable1_1 </ci>
+            </math>
+        </dataGenerator>
+    </listOfDataGenerators>
+    <listOfOutputs>
+        <plot2D id="plot1">
```
Comment (hsorby) [inline HumanSAN_Fabbri_Fantini_Wilders_Severi_2017.sedml:57]: Do we need to plot out anything for this container.

**Pull request merged**